ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+75.8 "Crumbling Crust (cast)"
+77.3 "Crumbling Crust (circles)"
+89.0 "Rush"
+91.1 "War Cry"
+93.8 "Earthquake"
+
+#~~~~~~~~#
+# BARDAM #
+#~~~~~~~~#
+
+# -ii 1EFE 2578 2579 257A 257E 257B
+
+1000.0 "Start" sync / 00:0839:.*Rebirth of Bardam the Brave will be sealed off/ window 1000,5
+1007.8 "Travail" sync /:Bardam:1EFF:/ window 7.8,5
+1014.1 "Magnetism" sync /:Hunter Of Bardam:1F08:/
+1020.3 "Tremblor" sync /:Bardam:257C:/
+1022.8 "Empty Gaze" sync /:Hunter Of Bardam:1F04:/
+1025.0 "Travail" sync /:Bardam:1EFF:/
+1029.8 "Charge x3" # sync /:Throwing Spear:257F:/
+1038.8 "Empty Gaze" sync /:Hunter Of Bardam:1F04:/
+1038.9 "Charge" sync /:Throwing Spear:257F:/
+1044.6 "--sync--" sync /:Bardam:24EA:/ window 15,15
+1052.3 "Travail" sync /:Bardam:1EFF:/
+1060.5 "Sacrifice" sync /:Bardam:1F01:/
+1060.7 "Travail" sync /:Bardam:1EFF:/
+1065.5 "Bardam's Ring" sync /:Bardam:2581:/
+1066.2 "Travail" sync /:Bardam:1EFF:/
+1076.4 "Comet x8" duration 12 # sync /:Bardam:257D:/
+1093.4 "Heavy Strike x3" sync /:Hunter Of Bardam:2577:/ duration 4
+1099.0 "Travail" sync /:Bardam:1EFF:/
+1105.3 "Comet Impact" sync /:Star Shard:2580:/
+1109.3 "--sync--" sync /:Bardam:24EA:/ window 15,15
+1116.9 "Travail" sync /:Bardam:1EFF:/
+1125.3 "Reconstruct" sync /:Bardam:1EFD:/
+1126.7 "--sync--" sync /:Star Shard:258B:/
+1128.3 "Travail" sync /:Bardam:1EFF:/
+1133.1 "Charge" sync /:Throwing Spear:257F:/
+1133.9 "Magnetism" sync /:Hunter Of Bardam:1F08:/
+1139.5 "Tremblor" sync /:Hunter Of Bardam:2585:/
+1140.0 "Heavy Strike x3" sync /:Warrior Of Bardam:2577:/ duration 4
+1153.0 "Meteor Impact" sync /:Looming Shadow:2582:/
+1153.4 "--sync--" sync /:Star Shard:258B:/
+
+#~~~~~#
+# YOL #
+#~~~~~#
+
+# -ii 1F0B 1F0E
+
+2000.0 "Start" sync / 00:0839:.*Voiceless Muse will be sealed off/ window 2000,5
+2008.1 "Feathercut" sync /:Yol:1F09:/
+2015.2 "Wind Unbound" sync /:Yol:1F0A:/
+2023.9 "Pinion" sync /:Yol Feather:1F11:/
+2025.8 "Flutterfall" sync /:Yol:1F0C:/
+2027.2 "Feathercut" sync /:Yol:1F09:/
+2033.9 "Pinion" sync /:Yol Feather:1F11:/
+2034.7 "Flutterfall" sync /:Yol:1F0C:/
+2042.2 "Wind Unbound" sync /:Yol:1F0A:/
+2046.4 "Feathercut" sync /:Yol:1F09:/
+2050.9 "Pinion" sync /:Yol Feather:1F11:/
+2054.0 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2059.2 "Feathercut" sync /:Yol:1F09:/
+2060.9 "Pinion" sync /:Yol Feather:1F11:/
+2069.3 "Wind Unbound" sync /:Yol:1F0A:/
+2071.4 "--untargetable--"
+2071.4 "--adds spawn--" sync /03:........:Added new combatant Corpsecleaner Eagle/ window 71.4,5
+2078.0 "Pinion" sync /:Yol Feather:1F11:/
+2088.0 "Pinion" sync /:Yol Feather:1F11:/
+2098.0 "Pinion" sync /:Yol Feather:1F11:/
+2108.0 "Pinion" sync /:Yol Feather:1F11:/
+2118.0 "Pinion" sync /:Yol Feather:1F11:/ jump 2078.0
+2128.0 "Pinion"
+2138.0 "Pinion"
+2148.0 "Pinion"
+
+# Unfortunately, because there are two Corpsecleaner Eagles,
+# we can't just sync to a 04 log line.
+2189.7 "--targetable--" sync /22:........:Yol:........:Yol:01/ window 120,5
```

Comment:
I know the eagles appear on a fixed time.  Does the boss coming back also appear on a fixed time, or is it relative to when the last eagle dies?

Reply:
The boss returns at the moment the second Eagle dies, or immediately after the animation for the charge attack completes, whichever is later. (I haven't tested extensively for whether there's a timer, but I did sit there for two minutes letting the first Eagle hit me, and nothing happened until the Eagle reached 50%, at which point Yol did its dive attack.)

It is in theory possible to have two intermission blocks, one for each Eagle, but that seemed like a bit too much just to avoid using a `22` line for the intermission close.